testsuite/features/secondary/srv_scc_user_credentials.feature

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: SCC user credentials in the Setup Wizard
+
+
+@no_mirror
+  Scenario: Enter some invalid organization credentials
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Setup Wizard > Organization Credentials"
+    And I ask to add new credentials
+    And I enter "SCC user" as "edit-user"
+    And I enter "SCC password" as "edit-password"
+    And I click on "Save"
+    Then I should see a "SCC user" text
+    And the credentials for "SCC user" should be invalid
+
+@no_mirror
+  Scenario: Make the credentials primary
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Setup Wizard > Organization Credentials"
+    And I make the credentials for "SCC user" primary
+    Then the credentials for "SCC user" should be primary
+
+@no_mirror
+  Scenario: Check the associated subscription list
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Setup Wizard > Organization Credentials"
+    And I view the subscription list for "SCC user"
+    And I wait until I see "No subscriptions available" text
+    And I click on "Close"
+
+# TODO
+# A test to edit the credentials is missing
+
+@no_mirror
+  Scenario: Cleanup: delete the new organization credentials
```

Comment:
Wow, really?
I thought we always had a SCC credential setup during our test suite. I wasn't aware we deleted it just after creation.
It would be good to double check this scenario don't cause any trouble.

Reply:
We indeed have one by default (at least when sumaform provides it): it is `UC7`.

This test adds a second one, which we immediately remove afterwards (idempotency).

Reply:
If you read the full feature, you will notice that we only create one (and only if we provide the parameter that activate the tag scc_credentials) and we delete this one, which is in fact the proper one, not a second fake one.

My guess is that is working, just because even if we remove them, as sumaform inject in a environment file on the server the scc_credentials, SUSE Manager take again this value from the file, and all still working.

Reply:
The real SCC credentials (`UC7`) are added via the step `I enter the SCC credentials` that reads the credentials from environment variables, in the case we have them. This was untouched by Alexei and it is fine so.

Then we have the code moved by Alexei. This code creates what I call the second user, and is not about `UC7` but about a so-called `SCC user`, added directly.

Both users are not the same and so it is safe to move it to secondary.

Or am I missing your point, @srbarrios?

Reply:
Yes, you are missing the ones added by sumaform in setup_env.sh
https://github.com/uyuni-project/sumaform/blob/master/salt/server/setup_env.sh#L17-L18

These are the ones I'm refering. If we remove or add some from the testsuite, it doesn't matter, unless we remove these variables from the setup_env.sh

Reply:
I don't miss them. These are the ones added by `I enter the SCC credentials` step, as mentioned above.

sumaform prepares the variables, and the test suite (the part untouched by Alexei) uses those variables to enter the UC7 credentials if they exist.

Reply:
Note the difference, I'm not talking about env. variables on the controller, but env. variables on the `server`.
The server when starts, read this variables, and automatically generate the SSC User.

Reply:
OK. I got it. We are creating a user "SCC User" for the test. And deleting exactly this "SCC User" account.

Reply:
Yes, exactly :-)